### PR TITLE
Blog App Milestone #4: Controllers set up

### PR DESCRIPTION
**Pull request by karlavdelgadof**
## Blog App Milestone #4: Controllers set up :hammer_and_wrench: :control_knobs:  :memo: 

These are the proposed changes :recycle:  for this project :arrow_down: :
- For each URL :link: created:
    - A route. :globe_with_meridians: 
    - An action in the correct controller. :heavy_check_mark: 
    - A view file. :page_facing_up: 
- Wrote placeholders written in plain HTML :100:

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
+  get '/users/:author_id/posts', to: 'posts#index'
+  get '/users/:author_id/posts/:id', to: 'posts#show'
+  get '/', to: "users#index"
+  get '/users/:id', to: "users#show"
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users
```
Comment (prolajumokeoni) [inline config/routes.rb]: - [x] Kindly protect your routes, do not expose unnecessary endpoints, According to the requirement, make sure that you added code to handle only the 4 required URLs which are users & posts index and show pages, see example below ⬇️ 

````ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
````

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @karlavdelgadof, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used 
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (karlavdelgadof) [reply config/routes.rb]: @prolajumokeoni Thank you, done :heavy_check_mark:

**Review by Davidosky007 — APPROVED** [review]
Hi @karlavdelgadof 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Proper use of Gitflow 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**